pynvme/nvme.i

```diff
@@ -18,6 +18,19 @@
 %rename(subsystem) nvme_subsystem;
 %rename(ns)        nvme_ns;
 
+/* There is a name conflict. In tree.h, the function nvme_ctrl_set_persistent()
+ * is already defined, which means that we can't define nvme_ctrl::set_persistent()
+ * when we extend "struct nvme_ctrl" (see "%extend nvme_ctrl" below).
+ * Instead, we define the method as nvme_ctrl::__set_persistent__() to avoid the
+ * conflict. Fortunately, we can work around the problem with SWIG's %rename()
+ * command so that in the generated Python code the method will be named
+ * set_persistent() and not __set_persistent__() as shown here:
+ *
+ *   def set_persistent(self, persistent: "bool") -> "void":
+ *       return _nvme.ctrl_set_persistent(self, persistent)
+ */
+%rename(set_persistent)  __set_persistent__;
```

Comment:
I'd rather change the name of the function, such that it doesn't clash with the SWIG generated names.
Easier, and one doesn't has to deal with the ugly rename construct.

Reply:
Ok. I renamed the function to `persistent_set()`